### ISSUE TITLE
malf AI and nuclear pinpointers

### ISSUE THREAD

**Issue by pedr69** [vgstation-coders/vgstation13#35407]
[ided][balance][discussion]
should they point to a delta AI even when it isn't shunted?
![image](https://github.com/vgstation-coders/vgstation13/assets/18052467/ebd8e7c3-ddeb-40f4-97ce-a00421572267)

**Comment by boy2mantwicethefam** [comment]
Yes I wanted to do that at some point but didn't get around to it. It's still up for grabs if anyone wants to code that.

**Comment by Securitymain** [comment]
no. It requires skill to have a borg drag you places without being seen. Reward taking risks
